charts/github-bot-safe-settings/README.md

```diff
@@ -0,0 +1,76 @@
+# safe-settings
+
+## `chart.deprecationWarning`
+
+## `chart.name`
+
+safe-settings
+
+## `chart.version`
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Saritasa | <nospam@saritasa.com> |  |
+
+## `chart.description`
+
+A GitHub App built with [Probot](https://github.com/probot/probot) to manage policy-as-code and apply repository settings to repositories across an organization.
```

Comment:
@darliiin could you leave here more comments about  "how can I add new rules and how does this bot scans repos?"

Reply:
Agree with @DanilaKazakevich please add more docs how it should be used and etc, what purpose of it. Take a look at our `tekton-apps` helm chart, it describes logics of a new app with examples and etc - https://github.com/saritasa-nest/saritasa-devops-helm-charts/blob/main/charts/tekton-apps/README.md